dp_mobility_report/model/overview.py

```diff
@@ -14,14 +14,19 @@
 
 
 def get_dataset_statistics(
-    mdreport: "MobilityDataReport", eps: Optional[float]
+    mdreport: "MobilityDataReport", eps: Optional[float], timestamps: bool
 ) -> Section:
     epsi = m_utils.get_epsi(mdreport.evalu, eps, 4)
 
     # counts for complete and incomplete trips
-    points_per_trip = (
-        mdreport.df.reset_index().groupby(const.TID).count()["index"].value_counts()
-    )
+    if timestamps:
```

Comment:
why should user IDs be counted instead of trip IDs if there are no timestamps?

Reply:
Was wrong. Therefore removed.